MaterialGraphProject/Assets/UnityShaderEditor/Editor/Data/Nodes/Artistic/Adjustment/ReplaceColorNode.cs

```diff
@@ -21,17 +21,15 @@ protected override MethodInfo GetFunctionToConvert()
             [Slot(1, Binding.None)] ColorRGB From,
             [Slot(2, Binding.None)] ColorRGB To,
             [Slot(3, Binding.None)] Vector1 Range,
-            [Slot(4, Binding.None)] out Vector3 Out)
```

Comment:
Got to preserve these IDs. Changing the ID for Out slot will break connections on existing graphs.